native/src/main/java/io/ballerina/stdlib/crypto/nativeimpl/Encrypt.java

```diff
@@ -116,4 +116,29 @@ public static Object encryptPgp(BArray plainTextValue, BString publicKeyPath, BM
             return CryptoUtils.createError("Error occurred while PGP encrypt: " + e.getMessage());
         }
     }
+
+    public static Object encryptPgpAsFile(BString inputFilePath, BString publicKeyPath, BString outputFilePath,
+                                          BMap options) {
+        byte[] publicKey;
+        try {
+            publicKey = Files.readAllBytes(Path.of(publicKeyPath.toString()));
+        } catch (IOException e) {
+            return CryptoUtils.createError("Error occurred while reading public key: " + e.getMessage());
+        }
+
+        try (InputStream publicKeyStream = new ByteArrayInputStream(publicKey);
+             InputStream inputStream = Files.newInputStream(Path.of(inputFilePath.toString()))
+        ) {
```

Comment:
her the formatting is different compared to L115. Better to make it consistent

Reply:
Will fix

Reply:
Fixed via [5ba88d7](https://github.com/ballerina-platform/module-ballerina-crypto/pull/567/commits/5ba88d7d21748a54d11ba4f7eda224128b2b288c)